src/pages/AddCar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../assets/App.css';
+import '../assets/addcar.css';
+
+const AddCar = () => (
+  <div className="content">
+    <div className="add-car">
+      <div className="add-form">
+        <h2>Add a car</h2>
+        <form>
+          <input type="text" placeholder="Brand ..." />
+          <input type="text" placeholder="Model ..." />
+          <input type="text" placeholder="Year ..." />
+          <button type="submit">Add car</button>
+        </form>
+      </div>
+      <div className="form-aside">
+        <img src="https://i.postimg.cc/y81zX370/car-photo-311956-removebg-preview.png" alt="aside" />
+      </div>
+    </div>
+  </div>
+);
```

Comment:
Please use useStates here like `const [model, setModel] = useState("");` and 
empty functions like `submit = (newCar) => console.log('newCar', newCar)` where newCar will be the object of every single car attributes including "color", "release_date", "description", etc... Please also add the remaining fields in the form.

If you did this before the backend is ready, when you have to implement with the backend for real, there won't be much left for you to code.